Assets/Prefabs/Enemies/BasicEnemy.prefab

```diff
@@ -231,7 +249,7 @@ SpriteRenderer:
   m_LightmapParameters: {fileID: 0}
   m_SortingLayerID: 0
   m_SortingLayer: 0
-  m_SortingOrder: 0
+  m_SortingOrder: 1
```

Comment:
I'd rather you not change the sprite sorting right now.  This is more easily done during final polish when we actually know what all the sprites are.